src/main/java/com/sourcery/gymapp/backend/globalConfig/GlobalCorsConfig.java

```diff
@@ -10,9 +10,9 @@ public class GlobalCorsConfig implements WebMvcConfigurer {
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/**")
-            .allowedOriginPatterns("http://localhost:3*")
-            .allowedMethods("GET", "POST", "PUT", "PATCH", "DELETE")
-            .allowedHeaders("*")
-            .allowCredentials(true);
+                .allowedOriginPatterns("http://localhost:3*")
+                .allowedMethods("GET", "POST", "PUT", "PATCH", "DELETE")
+                .allowedHeaders("*")
+                .allowCredentials(true);
     }
-}
+}
```

Comment:
Please try not to commit such files within the PR, if nothing changed 
+ always try to add new line in the end of a file
(otherwise it could introduce some minor inconveniences in version control, collaboration, and adherence to coding standards)

Reply:
I guess I'll need to manage more manual way to add files, usually I just do `git add .` which just adds all files. I see why it happened and can explain it in meeting

Reply:
TBH, I also use `git add . `
So we have little different Intellij Idea settings